src/textual/widget.py

```diff
@@ -327,54 +329,115 @@ def scroll_to(
 
         else:
             if x is not None:
-                self.scroll_target_x = self.scroll_x = x
                 if x != self.scroll_x:
+                    self.scroll_target_x = self.scroll_x = x
                     scrolled_x = True
             if y is not None:
-                self.scroll_target_y = self.scroll_y = y
                 if y != self.scroll_y:
+                    self.scroll_target_y = self.scroll_y = y
                     scrolled_y = True
-            self.refresh(repaint=False, layout=True)
+            if scrolled_x or scrolled_y:
+                self.refresh(repaint=False, layout=True)
+
         return scrolled_x or scrolled_y
 
-    def scroll_home(self, animate: bool = True) -> bool:
+    def scroll_relative(
+        self,
+        x: float | None = None,
+        y: float | None = None,
+        *,
+        animate: bool = True,
+    ) -> bool:
+        """Scroll relative to current position.
+
+        Args:
+            x (int | None, optional): X coordinate (column) to scroll to, or ``None`` for no change. Defaults to None.
+            x (int | None, optional): Y coordinate (row) to scroll to, or ``None`` for no change. Defaults to None.
+            animate (bool, optional): Animate to new scroll position. Defaults to False.
+
+        Returns:
+            bool: True if the scroll position changed, otherwise False.
```

Comment:
```suggestion
            x (int | None, optional): X distance (columns) to scroll, or ``None`` for no change. Defaults to None.
            y (int | None, optional): Y distance (rows) to scroll, or ``None`` for no change. Defaults to None.
            animate (bool, optional): Animate to new scroll position. Defaults to False.

        Returns:
            bool: True if the scroll position changed, otherwise False.
```